xbmc/windowing/gbm/DRMAtomic.cpp

```diff
@@ -64,8 +84,15 @@ void CDRMAtomic::DrmAtomicCommit(int fb_id, int flags, bool rendered, bool video
       AddProperty(m_gui_plane, "SRC_H", m_height << 16);
       AddProperty(m_gui_plane, "CRTC_X", 0);
       AddProperty(m_gui_plane, "CRTC_Y", 0);
-      AddProperty(m_gui_plane, "CRTC_W", m_mode->hdisplay);
-      AddProperty(m_gui_plane, "CRTC_H", m_mode->vdisplay);
+      if (CServiceBroker::GetWinSystem()->UseScalingFilter()){
```

Comment:
As this is specific to GBM windowing I wouldn't call into winsystem for this and just make it specific to CDRMAtomic